src/main/webapp/components/MockSearchButton.vue

```diff
@@ -46,6 +49,97 @@ module.exports = {
           isFavorited: false,
           pdfSource: "/assets/notes.pdf"
         },
+        {
+          thumbnailSrc: "/assets/notes.png",
+          author: "Andres H.",
+          title: "ABC set of notes",
+          dateCreated: new Date(Date.now() - 1000 * 60 * 60 * 24 * 6).toDateString(),
```

Comment:
prefer extracting this kind of math out into a constant (e.g. `SIX_DAYS` or something like `HOURS_PER_DAY * 6` if you'll be using this same constant in different ways) so it's clear how Date is being manipulated

Reply:
You got it!